contracts/facets/DFMoveFacet.sol

```diff
@@ -109,6 +113,13 @@ contract DFMoveFacet is WithStorage {
         }
 
         _executeMove(args);
+        
+        if(arenaStorage().startTime == 0) {
```

Comment:
I'd be curious how much gas these two code chunks are adding per move. If you can calculate that in a test that would be good to know.

Reply:
how would u recommend i do this?

Reply:
23191